sdk/identity/identity/src/credentials/interactiveBrowserCredential.browser.ts

```diff
@@ -47,6 +47,7 @@ export class InteractiveBrowserCredential implements TokenCredential {
   constructor(
     options: InteractiveBrowserCredentialInBrowserOptions | InteractiveBrowserCredentialNodeOptions
   ) {
+    options.loggingOptions?.allowLoggingAccountIdentifiers;
```

Comment:
I don't see how this value gets used?

Reply:
It's used under identityClient

Reply:
nevermind, I think i was just checking if the options existed.